extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -300,10 +300,26 @@ oppia.directive('imageWithRegionsEditor', [
         $scope.getCursorStyle = function() {
           return ($scope.regionDrawMode) ? 'crosshair' : 'default';
         };
-
         $scope.resetEditor = function() {
-          $scope.$parent.value.imagePath = '';
-          $scope.$parent.value.labeledRegions = [];
+          $modal.open({
+            templateUrl: 'modals/clearImageRegions',
+            backdrop: true,
+            controller: [
+              '$scope', '$modalInstance', function($scope, $modalInstance) {
+                $scope.confirmClear = function() {
+                  $modalInstance.close();
+                  alertsService.clearWarnings();
+                };
+                $scope.cancel = function() {
+                  $modalInstance.dismiss('cancel');
+                  alertsService.clearWarnings();
+                };
+              }
+            ]
+          }).result.then(function() {
+            $scope.$parent.value.imagePath = '';
```

Comment:
I'm wondering whether this line should be kept. From a user experience perspective, I think it seems a lot more intuitive to stay on the same image after clearing the regions, rather than going back to the list of images. Second opinion on this? @seanlip